pandas/tests/window/test_rolling.py

```diff
@@ -1706,7 +1709,11 @@ def test_rolling_quantile_interpolation_options(quantile, interpolation, data):
     if np.isnan(q1):
         assert np.isnan(q2)
     else:
-        assert q1 == q2
+        if not IS64:
+            # Less precision on 32-bit
+            assert np.allclose([q1], [q2])
```

Comment:
Any reason not to use `assert_allclose`?

Reply:
Do you mean the one under ``numpy.testing``?

Reply:
That is the one.

Reply:
Looks like np.testing is banned by a pre-commit hook. I've modified rtol and atol in ``np.allclose`` to match assert_allclose instead.